test-case/check-audio-equalizer.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+##
+## Case Name: check-audio-equalizer.sh
+## Preconditions:
+##    SOF Topology should have EQ component(s) in any pipeline
+## Description:
+##    check IIR/FIR config working
+## Case step:
+##    1. TODO: Check if the topology has a EQ component in it
```

Comment:
Could you make this done? I think a pre-check is very import here.
@aiChaoSONG Could you sync with @fredoh9 with your PR https://github.com/thesofproject/sof-test/pull/10 to support IIR/FIR comp

Reply:
@aiChaoSONG I'd like to add a feature to search specific DAPM name in tplgtool.py. 

1. The tool print all the name in the graph and i can grep it from the output
2. We can add cmdline parameter to search and match the name

I'm fine with either one. I'd like to stay in sync with PGA support. However I'm not getting the idea of your PR #10. How are you going to support PGA name?

Reply:
@aiChaoSONG  I think you also need modify sof-tplgreader.py file to add this filed

Reply:
@fredoh9  each widget is represented by a node structure, the fields of a node are:
- prev: previous node
- next: next node
- name: the name of the node.(in fact, each widget may have two names: the name field and the sname field. both names may be used in the graph from tplg, so the name here is from graph, it may be `name` or `sname` of a widget)
- widget: the widget bind with this node.
- ctrl: kcontrol between two widgets, currently, we don't use this, because all kcontrol is "" in the graph from tplg.

_link_graph function links the graph according to the tplg graph info. it returns two structures:
- node_list: a list of all nodes of this tplg. the node fileds are assigned by traverse the tplg graph, so we have the graph already.
- link_head_list: for each pcm, we have an independent graph, this list only store the head node. (this structure is optional, it is used to separate each pipeline) 

the find_connected_comp is used to find the connected component for a specified component, eg. find PGA component for specified PCM. it use reference node and the component type you want to find for ref node as it's arguments. the second parameter is a string. because you can see that all PGA  names is started with "PGA",and  buffers with "BUF", etc.

to use the feature, first you should have a component name,  and you find the node with `find_node_by_name`, and use `find_connected_comp` to find the connected nodes you want, the 'widget' filed of node is the info you need.

Reply:
@fredoh9  As I can see, the EQ name is started with "EQ",  and you should have a PCM name, eg PCM0P. with `find_node_by_name`, you got the node for PCM0P, with `find_connected_comp(pcm_node, "EQ")`, you got all the EQ component connected to PCM0P.
if EQIIR and EQFIR matters to you, you can call `find_connected_comp` twice, one with "EQFIR"  and the other with "EQFIR". 
*Note: the return value of `find_connected_comp` is a list.   the "prev" of head node is "None", and the "next" field of tail node is "None", that's how the finding process works.*

Reply:
@fredoh9 @xiulipan @Bin-QA  after #10 merged, I will modified sof-tplgreader.py, and output EQs and PGAs for each PCM, just like rate and format, I think this is better.